docs/chat/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "RQ Discord"
+layout: chat
+---
+
+Join our discord [here](/chat/join) if you need help or want to chat about contributions or what should come next in RQ.
+
+<iframe src="https://discord.com/widget?id=844816706231861248&theme=dark" height="500" allowtransparency="true" frameborder="0" sandbox="allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts" style="width: 100%;"></iframe>
```

Comment:
Sorry for being nitpicky, can we eliminate the use of iframe and JS here? A simple `href` that opens in a new tab will do.

Reply:
This iframe is a widget I copied from discord. It is not intended for a new tab. It is not so important so I can delete it. Shall I?

Reply:
Yes please, you can also just turn there `[here]` text into a hyperlink so we can get rid of `docs/chat/join.html`.

Reply:
Ok, I made `chat/join` so there is a readable link for the discord, but I remove it.